### PR TITLE
ROC-5350 rpa new judgment types

### PR DESCRIPTION
**Pull request by justin-r-oliver**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5350

### Change description ###

1. Add judgment by admission and default judgment to the robot's code.
2. Rename judgment email's subject line to "J judgment request [case num]"
3. Remove Sonar checks for rpa-mapper (Sonar incorrectly reports zero test coverage)

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5145313&bid=16900474)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5145313&bid=16900474)

File: src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgementNotificationService.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ private EmailData prepareEmailData(Claim claim) {
         EmailAttachment ccjPdfAttachment = generateCountyCourtJudgmentPdf(claim);
 
         return new EmailData(emailProperties.getCountyCourtJudgementRecipient(),
-            "J default judgement request " + claim.getReferenceNumber(),
+            "J judgement request " + claim.getReferenceNumber(),
```
Comment (andrewwa-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgementNotificationService.java]: This should be "J judgment request [case num]" as per A/C.
Please also correct the spelling of "Judgment" (judgement is the US spelling) in any new code / files. (Also renaming existing files if you fancy.)

File: rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java [hunk 2]
```diff
@@ -29,5 +31,17 @@ public JsonObject map(Claim claim) {
...
             .build();
     }
+
+    private String mapCountyCourtJudgmentType(CountyCourtJudgmentType ccjType) {
+
+        switch (ccjType) {
+            case ADMISSIONS:
+                return "BY_ADMISSION";
+            case DEFAULT:
+            case DETERMINATION:
```
Comment (andrewwa-kainos) [inline rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java]: Determination is not supported - I would expect this to throw an exception

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java  3
- src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgmentNotificationService.java  5
- src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgmentNotificationServiceTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5145313&bid=16900474)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java  3
- src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgmentNotificationServiceTest.java  1
- src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgmentNotificationService.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5145313&bid=16900474)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgmentJsonMapper.java  4
- rpa-mapper/src/test/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgmentJsonMapperTest.java  1
- src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgmentNotificationService.java  5
- src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgmentNotificationServiceTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5145313&bid=16900474)

**Comment by justin-r-oliver** [comment]
Merge after the RPA team have released their corresponding changes.

**Pull request closed without merging**